kong/plugins/opentelemetry/utils.lua

```diff
@@ -1,9 +1,17 @@
 local http = require "resty.http"
 local clone = require "table.clone"
-
+local sandbox = require "kong.tools.sandbox"
+local deep_copy = require("kong.tools.table").deep_copy
```

Comment:
```suggestion
local cycle_aware_deep_copy = require("kong.tools.table").cycle_aware_deep_copy
```

sorry, I noticed this last-minute